src/components/Form/Radiobutton.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { HtmlInput, HtmlLabel, HtmlSpan, HtmlDiv } from '../../reset';
+import { idGenerator } from '../../utils/uuid';
+import { logger } from '../../utils/logger';
+import {
+  RadiobuttonGroupConsumer,
+  RadiobuttonGroupProviderState,
+} from './RadiobuttonGroup';
+
+const baseClassName = 'fi-radiobutton';
+
+const radiobuttonClassNames = {
+  container: `${baseClassName}_container`,
+  input: `${baseClassName}_input`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
+};
+
+type RadiobuttonVariant = 'small' | 'large';
+
+export interface RadiobuttonProps {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /**
+   * Label element content
+   */
+  children?: ReactNode;
+  /** Group multiple Radiobuttons to be inside same selection group */
+  name?: string;
```

Comment:
There is a small chance for error when RadiobuttonGroup name is defined and name is also provided for the button as these can be different. Or does the name on the group have any effect on things?

Reply:
I tested it now on the styleguidist and the `name` prop that **RadiobuttonGroup** do give will be used, as it happens later.

Reply:
Might be worth documenting this to the radioGroup/radiobutton. E.g. overriden by group name